### ISSUE TITLE
Undefined symbols for architecture i386

### ISSUE THREAD

**Issue by BenjaminDieter** [mixpanel/mixpanel-iphone#582]
Hi guys. I'm writing an Ionic app and want to utilize Mixpanel. I'm using [cordova-plugin-mixpanel](https://github.com/samzilverberg/cordova-mixpanel-plugin). Since I'm developing on a Windows machine I'm also using TACO for remote builds.

Trying to build with `taco build ios`, I get the following error message:

> Undefined symbols for architecture i386:
  "\_OBJC\_CLASS\_$\_WKInterfaceDevice", referenced from:
      objc-class-ref in MixpanelWatchOS.o
ld: symbol(s) not found for architecture i386
clang: error: linker command failed with exit code 1 (use -v to see invocation)

> ** BUILD FAILED **

The guys from `cordova-plugin-mixpanel` [told me](https://github.com/samzilverberg/cordova-mixpanel-plugin/issues/41) to ask here.

Any ideas on how to resolve this issue?

**Comment by yarneo** [comment]
Hey @BenjaminDieter, the file you are getting an error on is only used with watchOS apps. It shouldn't really be part of your target if you are just implementing for the iPhone. Therefore what I would suggest is removing that file entirely. [cordova-plugin-mixpanel](https://github.com/samzilverberg/cordova-mixpanel-plugin) seems to have added that file to their repo, you can try and fork it and remove the file and see how that goes. Let me know if I can help further.

**Comment by BenjaminDieter** [comment]
Hey @yarneo, first of all thanks for answering so fast. Unfortunately, deleting the `MixpanelWatchOS.o` did not bring the desired result. I checked thrice that there is no `MixpanelWatchOS.o` existent, nevertheless XCode still throws the error. Neither XCode nor Finder are able to find the file. I also deleted the lines containing `MixpanelWatchOS.o` from the LinkFileList.

Anyways, deleting the contents of `MixpanelWatchOS.h`, `MixpanelWatchOS.m` and `watchos.modulemap` (resulting in no `MixpanelWatchOS.o` being generated) did the trick for me. Thanks!
